hypershift-operator/main.go

```diff
@@ -399,8 +399,11 @@ func run(ctx context.Context, opts *StartOptions, log logr.Logger) error {
 		}
 	}
 
-	if err := hostedclustersizing.SetupWithManager(ctx, mgr, operatorImage, releaseProviderWithOpenShiftImageRegistryOverrides, imageMetaDataProvider); err != nil {
-		return fmt.Errorf("failed to set up hosted cluster sizing operator: %w", err)
+	enableClusterSizing := os.Getenv("ENABLE_HOSTED_CLUSTER_SIZING") == "1"
+	if enableClusterSizing {
+		if err := hostedclustersizing.SetupWithManager(ctx, mgr, operatorImage, releaseProviderWithOpenShiftImageRegistryOverrides, imageMetaDataProvider); err != nil {
```

Comment:
You need to also gate the placeholder deployment controllers on this, and perhaps pass the value down to the HCCO and disable the HCP status nodecount controller?

Reply:
Thx! done